loopin.go

```diff
@@ -1062,6 +1118,22 @@ func (s *loopInSwap) publishTimeoutTx(ctx context.Context,
 	return fee, nil
 }
 
+// setStateAbandoned stores the abandoned state in the database and announces
+// it.
+func (s *loopInSwap) setStateAbandoned(ctx context.Context) error {
+	if s.state.IsPending() {
```

Comment:
Does it make sense to abandon the swap is the htlc is already published? Maybe we should instead create a function whether the swap is "abandonable".

Reply:
For our resolution of an issue the swap is caught in a pending state although the swap itself hasn't been published, so there it makes sense to abandon a swap in "published" state.